program/src/processor.rs

```diff
@@ -5755,6 +5833,28 @@ impl Processor {
                 msg!("Mango: SetDelegate");
                 Self::set_delegate(program_id, accounts)
             }
+            MangoInstruction::ChangeSpotMarketParams {
+                maint_leverage,
+                init_leverage,
+                liquidation_fee,
+                optimal_util,
+                optimal_rate,
+                max_rate,
+                version,
+            } => {
+                msg!("Mango: ChangeSpotMarketParams");
+                Self::change_spot_market_params(
+                    program_id,
+                    accounts,
+                    maint_leverage,
+                    init_leverage,
+                    liquidation_fee,
+                    optimal_util,
+                    optimal_rate,
+                    max_rate,
+                    version,
```

Comment:
another thing for a cleanup/refactoring: passing these as arguments is error prone. It'd be nice if `ChangeSpotMarketParams` was a struct that could just be passed onwards to the executor function

Reply:
I will move this to the refactoring ticket, noted.